files/galaxy/dynamic_job_rules/production/total_perspective_vortex/users.yml.j2

```diff
@@ -309,5 +309,8 @@ users:
       params:
         object_store_id: minio_test
     selenium_st_aws_gateway@genome.edu.au:
+      context:
+        slurm_singularity_volumes: "{{ slurm_singularity_volumes }},/mnt/aws_gateway:ro"
```

Comment:
Why is this read-only though? Shouldn't it be writable for output files?

Reply:
write is not required for any of the other user data folders. galaxy moves the files there after the job has run. see volumes listed in host_vars/galaxy

Reply:
Right! I forgot the handler does the final move.